### DISCUSSION TITLE
Deploy Next js To Lambda

### DISCUSSION THREAD

**Discussion by danielcristho** [awsugid/forum#28] [Q&A]
Apakah dengan mendeploy frontend ke Lambda termasuk best practice? Atau ada pilihan lain misalnya deploynya ke Amplify. Untuk frontendnya saya menggunakan Next js. Mohon sarannya.

**Comment by iilness2** [comment]
dpeloy nextjs ke s3 bisa juga. 

Untuk caranya pernah tak post disini: https://dev.to/iilness2/practical-approach-deploy-nextjs-with-s3-4k3l

**Reply by danielcristho** [reply]
Oke terima kasih sarannya mas.
